pkg/validator/output.go

```diff
@@ -62,11 +59,12 @@ type ResultSet map[string]ResultMessage
 
 // ControllerResult provides results for a controller
 type ControllerResult struct {
-	Name      string
-	Namespace string
-	Kind      string
-	Results   ResultSet
-	PodResult PodResult
+	Name        string
+	Namespace   string
+	Kind        string
+	Results     ResultSet
+	PodResult   PodResult
+	CreatedTime time.Time
```

Comment:
Why did this get added?

Reply:
The time was there specifically for the dedupe logic